Tasks/PackerBuild/src/taskParameters.ts

```diff
@@ -59,6 +61,7 @@ export default class TaskParameters {
                 this.deployScriptArguments = tl.getInput(constants.DeployScriptArgumentsInputName, false);
             }                
 
+            this.additionalBuilderParameters = JSON.parse(tl.getInput("additionalBuilderParameters"));
```

Comment:
improved error message. Infact added better error messages for other input handling as well.